scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -18,7 +18,6 @@ local GAME_TIE = 3;
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
     if (npcUtil.tradeHas(trade, nil, 5)) then
```

Comment:
npcUtil is filled with a lot of garbage nobody has ever or will ever willingly use except the guy who created them. So I vote to yes use trade:getGil() as it makes tons more sense.